### ISSUE TITLE
Auto log formatter

### ISSUE THREAD

**Issue by ForNeVeR** [codingteam/loglist#86]
There should be a (maybe semi-sentient) being for log formatting embedded to the posting form. IT should automatically resolve nicknames and put them into angle braces. It should also remove timestamps.

**Comment by rexim** [comment]
I agree that we don't need timestamps in general. But they may be essential for some quotes. So this part should configurable. Also I think such formatter should exist on the approver's. It should work before the quote is approved so the approver has a chance to change something manually.

**Comment by ForNeVeR** [comment]
I was thinking about adding a "format shit" button near the "approve shit" one.

**Comment by rexim** [comment]
Yeah, that's what I'm talking about! :)
